server/src/routes/csvFileRoutes.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { convertCSV } from '../controller/csvController'; // Import csv controller
+import { upload } from '../middleware/uploadMiddleware'; // Import the upload middleware
+
+const router = express.Router();
+
+// Define the POST route for CSV conversion, using the upload middleware to handle file uploads
+router.post('/convert-csv', upload.single('file'), convertCSV);
```

Comment:
any reason why you are using another whole file just for csv? maybe use the existing fileRoutes.

Reply:
the comment above still applies to this file.